test_package/conanfile.py

```diff
@@ -2,25 +2,14 @@
 
 
 class DummyTestConan(ConanFile):
-    settings = "os"
+    settings = "os", "arch", "compiler", "build_type"
     generators = "cmake_find_package"
-    requires = "dummy/1.0@demo/testing"
-
-    def configure_cmake(self):
-        cmake = CMake(self)
-        cmake.configure()
-
-        return cmake
 
     def build(self):
         self.cmake = CMake(self)
-
         print("{}".format(self.cmake.command_line))
-
         self.cmake.configure()
         self.cmake.build()
 
-
     def test(self):
-        self.cmake.test()
-
+        pass
```

Comment:
here you should test the binary generated by this example, normally executing the binary and checking the output or exit code